### PR TITLE
WordPress: add some filters to customize behavior

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Add some filters in key places, so make it easier to customize this plugin.

## Testing Instructions

---

Fixes #

### CODE REVIEW

File: includes/Traits/Publisher.php [hunk 1]
```diff
@@ -40,7 +40,15 @@ trait Publisher {
...
-		return WEBSTORIES_PLUGIN_DIR_URL . 'assets/images/fallback-wordpress-publisher-logo.png';
+		$placeholder_logo_url = WEBSTORIES_PLUGIN_DIR_URL . 'assets/images/fallback-wordpress-publisher-logo.png';
+		/**
+		 * Filters the publisher's logo placeholder.
+		 *
+		 * This should point to a square image.
+		 *
+		 * @param string $placeholder_logo_url URL to the publisher's logo placeholder.
+		 */
+		return apply_filters( 'publisher_logo_placeholder', $placeholder_logo_url );
```
Comment (swissspidy) [inline includes/Traits/Publisher.php]: I think I've mentioned in on Slack before, but I don't really see a need for this filter.

What does this achieve that `web_stories_publisher_logo` doesn't already cover?

`web_stories_publisher_logo` allows overriding the logo and also allows providing a fallback if no logo is set

Comment (spacedmonkey) [reply includes/Traits/Publisher.php]: Okay, maybe we could add the placeholder to the `web_stories_publisher_logo` for context?

Comment (swissspidy) [reply includes/Traits/Publisher.php]: As an additional param to the filter?

Comment (spacedmonkey) [reply includes/Traits/Publisher.php]: > As an additional param to the filter?

Yes

Comment (swissspidy) [reply includes/Traits/Publisher.php]: Yeah we can do that

Comment (spacedmonkey) [reply includes/Traits/Publisher.php]: Fixed https://github.com/google/web-stories-wp/pull/2667/commits/2eeee5220a340a226e87a822d3aaaeac3281ab73

**Review by swissspidy — APPROVED** [review]

**Pull request merged**